src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File file = new File(fileName);
+        try {
+            OutputStream byteStream = new FileOutputStream(file);
+            byteStream.write(data);
+            byteStream.close();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't write to file " + fileName);
```

Comment:
let's add the original exception as the cause
```suggestion
            throw new RuntimeException("Can't write to file " + fileName, e);
```

Reply:
fixed